ws-nextjs-app/pages/[service]/send/[id]/FormField/styles.ts

```diff
@@ -33,53 +36,94 @@ export default {
-      ...fontVariants.sansRegular,
-      ...fontSizes.pica,
-    }),
-  checkboxContainer: ({ spacings }: Theme) =>
+  textField:
+    (useErrorTheme: boolean) =>
+    ({ spacings, fontVariants, fontSizes, palette }: Theme) =>
+      css({
+        border: `solid 0.0625rem ${useErrorTheme ? palette.ERROR_CORE : palette.GREY_10}`,
+        outline: 'solid 0.0625rem transparent',
+        width: '100%',
+        minHeight: `2.75rem`,
+        padding: `${spacings.FULL}rem`,
+        ...fontVariants.sansRegular,
+        ...fontSizes.pica,
+      }),
+  checkboxContainer: () =>
     css({
       display: 'flex',
       flexWrap: 'nowrap',
-      paddingBottom: `${spacings.DOUBLE}rem`,
     }),
+  maxWordLabel:
+    (useErrorTheme: boolean) =>
+    ({ palette }: Theme) =>
+      css({
+        marginBottom: `${pixelsToRem(6)}rem`,
+        color: useErrorTheme ? palette.ERROR_CORE : palette.GREY_10,
+      }),
   textArea: () =>
     css({
       resize: 'none',
+      display: 'block',
     }),
   checkboxLabel: ({ spacings }: Theme) =>
     css({
       flex: 'auto',
       marginInlineStart: `${spacings.DOUBLE}rem`,
       cursor: 'pointer',
     }),
-  checkbox: ({ palette, spacings }: Theme) =>
-    css({
-      display: 'inline-block',
-      flex: 'initial',
-      flexShrink: 0,
-      width: `${pixelsToRem(30)}rem`,
-      height: `${pixelsToRem(30)}rem`,
-      cursor: 'pointer',
-      boxSizing: 'border-box',
-      border: `solid 0.0625rem ${palette.GREY_10}`,
-      marginBottom: `${spacings.FULL}rem`,
-      appearance: 'none',
-      '&:checked::after': {
-        content: '""',
-        backgroundImage: `url("${CHECK_IMG}")`,
-        backgroundRepeat: 'no-repeat',
-        backgroundPosition: '50%',
-        backgroundSize: '0.889rem',
+  checkbox:
+    (useErrorTheme: boolean) =>
+    ({ palette }: Theme) =>
+      css({
         display: 'inline-block',
-        width: '100%',
-        height: '100%',
+        flex: 'initial',
+        flexShrink: 0,
+        width: `${pixelsToRem(30)}rem`,
+        height: `${pixelsToRem(30)}rem`,
+        cursor: 'pointer',
+        boxSizing: 'border-box',
+        border: `solid 0.0625rem ${useErrorTheme ? palette.ERROR_CORE : palette.GREY_10}`,
+        appearance: 'none',
+        '&:checked::after': {
+          content: '""',
+          backgroundImage: `url("${CHECK_IMG}")`,
+          backgroundRepeat: 'no-repeat',
+          backgroundPosition: '50%',
+          backgroundSize: '0.889rem',
+          display: 'inline-block',
+          width: '100%',
+          height: '100%',
+        },
+      }),
+  errorText: ({ palette }: Theme) =>
+    css({
+      color: palette.WHITE,
+    }),
+  errorSvg: ({ mq, palette }: Theme) =>
+    css({
+      fill: palette.WHITE,
+      verticalAlign: 'middle',
+      marginInlineEnd: '0.75rem',
+      [mq.HIGH_CONTRAST]: {
+        path: {
+          fill: 'currentColor',
+        },
       },
     }),
+  errorMessageBox: ({ palette }: Theme) =>
```

Comment:
I think we need to add in some sort of border for this box and for the error arrow for high contrast mode? (We can use the HIGH_CONTRAST media query if adding in borders affect the UX).

![Screenshot 2024-06-18 at 10 26 26](https://github.com/bbc/simorgh/assets/98817636/9a293a05-a069-4be1-917e-58aca9f3d145)